pyproject.toml

```diff
@@ -0,0 +1,81 @@
+[tool.poetry]
+name = "python_dwd"
+version = "0.1.0"
+description = "Python Library For Accessing Weather Data From German Weather Service"
+authors = [
+    "Benjamin Gutzmann <gutzemann@gmail.com>",
+    "Daniel Lassahn <daniel.lassahn@meteointelligence.de>",
+    "Andreas Motl <andreas.motl@panodata.org>"
+]
+license = "MIT"
+readme = "README.md"
+include = ["LICENSE.md", "CHANGELOG.md", "CODE_OF_CONDUCT.md", "DWD_FTP_STRUCTURE.md"]
+homepage = "https://earthobservations.github.io/python_dwd/"
+repository = "https://github.com/earthobservations/python_dwd"
+keywords = ["dwd", "weather", "observations", "forecasts", "weather-data", "open-data"]
+classifiers = [
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT",
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: System Administrators",
+    "Topic :: Communications",
+    "Topic :: Database",
+    "Topic :: Internet",
+    "Topic :: Internet :: WWW/HTTP :: Indexing/Search",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Topic :: Scientific/Engineering :: GIS",
+    "Topic :: Scientific/Engineering :: Human Machine Interfaces",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "Topic :: Scientific/Engineering :: Interface Engine/Protocol Translator",
+    "Topic :: Scientific/Engineering :: Visualization",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: System :: Archiving",
+    "Topic :: Text Processing",
+    "Topic :: Utilities",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Operating System :: MacOS"
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+pandas = "1.0.4"
+tables = "3.6.1"
+numpy = "1.18.3"
+scipy = "1.4.1"
+h5py = "2.10.0"
+cachetools = "^3.1.1"
+pathlib = "^1.0.1"
```

Comment:
Do we have to define python internal packages here?

Reply:
Probably you are speaking about pathlib? I accidentially had put that standard library in the requirements, but obviously we can omit this one.

Reply:
I've just removed `pathlib`. Thanks for spotting this!